### PR TITLE
A0-2751: make errors punctuation consistent

### PR DESCRIPTION
**Pull request by youPickItUp**

### CODE REVIEW

File: packages/extension-ui/src/Popup/Export.tsx [hunk 1]
```diff
@@ -62,12 +62,12 @@ function Export({
       })
       .catch((error: Error) => {
         console.error(error);
-        setError(error.message);
+        setIsProvidedPassWrong(true);
```
Comment (jalooc) [inline packages/extension-ui/src/Popup/Export.tsx:65]: Not sure if _wrong password_ is the only kind of error that can be caught here (and the other similar spots in this PR). Just rising a concern, no stress about it.

File: packages/extension-ui/src/components/PasswordField/getFeedback.ts [hunk 2]
```diff
@@ -16,23 +16,39 @@ const APP_RELATED_INPUTS = ['aleph', 'zero', 'aleph zero', 'signer'];
...
-  ];
+  return [...APP_RELATED_INPUTS, ...nameRelatedInputs];
 };
 
-export const isPasswordTooWeak = (feedback: ValidationResult) =>  {
+export const isPasswordTooWeak = (feedback: ValidationResult) => {
   return feedback.score < LOWEST_ACCEPTED_PASSWORD_SCORE;
 };
 
+const nonLatinAlphanumeric = /\W/;
```
Comment (jalooc) [inline packages/extension-ui/src/components/PasswordField/getFeedback.ts:26]: Kudos for a const for that 👍

File: packages/extension-ui/src/components/PasswordField/getFeedback.ts [hunk 3]
```diff
@@ -16,23 +16,39 @@ const APP_RELATED_INPUTS = ['aleph', 'zero', 'aleph zero', 'signer'];
...
+
+const addPunctuationIfMissing = (sentence: string) => {
+  const lastChar = sentence.at(-1);
+
+  if (!lastChar) {
```
Comment (jalooc) [inline packages/extension-ui/src/components/PasswordField/getFeedback.ts:31]: When would the last char be falsy? Does that basically mean the `sentence` is an empty string?

File: packages/extension-ui/src/components/PasswordField/getFeedback.ts [hunk 4]
```diff
@@ -16,23 +16,39 @@ const APP_RELATED_INPUTS = ['aleph', 'zero', 'aleph zero', 'signer'];
...
+    return sentence;
+  }
+
+  if (lastChar.match(nonLatinAlphanumeric)) {
```
Comment (jalooc) [inline packages/extension-ui/src/components/PasswordField/getFeedback.ts:35]: What sentence endings other than `.`, `!` or `?` should stop us from appending a dot?

**Review by jalooc — APPROVED** [review]
I like that you went with a dot at the end of each sentence - proper grammar :D

File: packages/extension-ui/src/components/PasswordField/getFeedback.ts (hunk 3 above)
Comment (youPickItUp) [reply packages/extension-ui/src/components/PasswordField/getFeedback.ts:31]: yes, it's mostly "typescript not complaining" thing

File: packages/extension-ui/src/components/PasswordField/getFeedback.ts (hunk 4 above)
Comment (youPickItUp) [reply packages/extension-ui/src/components/PasswordField/getFeedback.ts:35]: I was a bit afraid of non-european languages in general. Perhaps it's an overkill :shrug: And basically any punctuation - but that assumes someone ends a sentance with `:` which shouldn't happen :shrug:

**Pull request merged**